.github/workflows/documentation.yml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - main
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: '1'
+      - name: Install ChaosTools.jl on main branch
+        shell: julia --color=yes --project=docs/ {0}
+        run: |
+          using Pkg
+          Pkg.add(PackageSpec(; name="ChaosTools", rev="main"))
```

Comment:
There should be an open issue reminding ourself that installing the main branch of ChaosTools should be reverted to standard installation from the General repo once all package conflicts are resolved